### PR TITLE
llvm3.8 build tutorial

### PR DESCRIPTION
**Pull request by kren1**
I modified the llvm3.4 tutorial a bit, to show how to build with llvm 3.8. It's still a bit rough, but I think it's important to start pushing for newer versions and this seems an easy way to start.

### CODE REVIEW

File: build-llvm38.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+   $ cd klee
+   ```
+
+   Get the relevant pull requests:
+
+   ```bash
+   $ 
+     git fetch origin pull/605/head:pull605 ;\
+     git fetch origin pull/729/head:pull729 ;\
+     git fetch origin pull/780/head:pull780
```
Comment (ccadar) [inline build-llvm38.md]: I merged this one, so you can remove it

File: build-llvm38.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+   Fix the trivial merge conflict in `test/Runtime/POSIX/DirSeek.c`, by removing the diff boilerplat lines : `47, 48, 51`.
+
+   ```bash
+   $ git add test/Runtime/POSIX/DirSeek.c
+   $ git commit
+   ```
+
+
+   ```bash
+   $ git merge pull780
```
Comment (ccadar) [inline build-llvm38.md]: same here

File: build-llvm38.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+   $ git commit
+   ```
+
+
+   ```bash
+   $ git merge pull780 
+   ```
+
+     git merge pull669 ;\
+   git fetch origin pull/669/head:pull669 ;\
```
Comment (ccadar) [inline build-llvm38.md]: Move this fetch with the others

**Comment by kren1** [comment]
Done, looks much better now. I still kept the fixing the conflict instructions, because the conflist still occurs. Also pull/669 is not needed because it's included in pull/605, so I removed it.

**Comment by ccadar** [comment]
Great, thanks!  Can you rebase this and squash the last two commits into the previous ones?

**Comment by kren1** [comment]
Done

**Comment by ccadar** [comment]
Thank, @kren1, I merged this, but I've just realized you haven't added a link to http://klee.github.io/getting-started/, can you do that?

**Comment by kren1** [comment]
I did add that link. I also see it online. Is it not working for you?

**Comment by ccadar** [comment]
You're right, my browser did not refresh that page for some reason.  Thanks again.

**Pull request merged**